django/db/migrations/writer.py

```diff
@@ -184,6 +185,9 @@ def as_string(self):
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
 
+        if self.migration.initial:
```

Comment:
`if self.migration.initial is not None` -- after all you set `False` explicitly.